docs/README.md

```diff
@@ -55,6 +55,8 @@ With a clear project scope, lightweight governance model, and clean lines of
 separation between pluggable components, the Knative project establishes an
 efficient contributor workflow.
 
+While not part of a software foundation, Knative provides contributors with a clear project scope, open governance with multiple companies represented, and clean lines of separation between pluggable components. The Knative project strives for an efficient contributor workflow.
```

Comment:
@lindydonna  There's an extra space in the front of the paragraph. Also, what do you think of this edited version:

"While not a part of a software foundation, Knative provides contributors with a clear project scope and an open governance model where multiple companies are represented. The Knative project strives for clean lines of separation between pluggable components and an efficient contributor workflow."

Reply:
I like the change, but would add a comma after components (or perhaps an 'as well as') so it doesn't seem like a comparison between components and workflow:

"While not a part of a software foundation, Knative provides contributors with a clear project scope and an open governance model where multiple companies are represented. The Knative project strives for clean lines of separation between pluggable components as well as an efficient contributor workflow."

Reply:
@jdumars I added "both" to separate the two phrases. A comma isn't grammatical there. :)

Reply:
Thanks!